src/main/scala/com/scality/clueso/CluesoConstants.scala

```diff
@@ -0,0 +1,35 @@
+package com.scality.clueso
+
+import org.apache.spark.sql.types._
+
+object CluesoConstants {
+
+  val sstSchema = new StructType()
+    .add("userMd", MapType(StringType, StringType, true), true)
```

Comment:
As discussed, I will modify the metadata in backbeat before it goes through kafka so by the time it arrives in spark, it will have this userMD object.